plugins/module_utils/data_set.py

```diff
@@ -1745,9 +1745,10 @@ def temp_member_name():
     """Generate a temp member name"""
     first_char_set = ascii_uppercase + "#@$"
     rest_char_set = ascii_uppercase + digits + "#@$"
-    temp_name = first_char_set[randint(0, len(first_char_set) - 1)]
-    for i in range(7):
-        temp_name += rest_char_set[randint(0, len(rest_char_set) - 1)]
+    # using sample as k=1 and k=7 to avoid using random.choice just for oneline import
```

Comment:
i'd add a comment about what the code currently does -- eg 1st line gets a char, next line gets 7 chars/digits